personal_assistant/pa_file/templates/file_list.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'pa_core/home.html' %}

```

Comment:
Добре, що зробив таке унаслідування але з часу цієї пропозиції зʼявився ще кращий варіант тож раджу скористатися вже ним (свій шаблон назвати так само для узгодженості назв):
```suggestion
{% extends 'pa_core/overview.html' %}

```